Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Docker image to run the tests in Jenkins.
+
+FROM ubuntu:17.04
+MAINTAINER development.team@moneyadviceservice.org.uk
+
+# Install needed packages.
+RUN apt-get update -q \
+    && DEBIAN_FRONTEND="noninteractive" apt-get upgrade \
+            -qy \
+            -o Dpkg::Options::="--force-confnew" \
```

Comment:
It means that if a package is upgraded that contains example configuration files as part of the package, install that configuration file if it has changed. Basically "give me the latest software and its default config files".